app/AppKernel.php

```diff
@@ -52,8 +52,8 @@ public function registerBundles()
             new Symfony\Bundle\MonologBundle\MonologBundle(),
             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
             new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
-            new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
             new ApiPlatform\Symfony\Bundle\ApiPlatformBundle(),
+            new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
```

Comment:
🤔  why did you do that ?

Reply:
It was during rebase, there was the line twice I delete the wrong line sweat_smile

Reply:
But now the order is different, I guess it has impact, doesn't it?

Reply:
It can have impacts if one bundle requires services from another. I don't think for these two bundle it can mess something. But as this change is not needed, it is safer IMO to remove it.